### ISSUE TITLE
[BUG] Password length message in login

### ISSUE THREAD

**Issue by Gauravtb2253** [ajaynegi45/Uttarakhand-Culture-NewUI#159]
### Is this a new issue?

- [X] I have searched "open" and "closed" issues, and this is not a duplicate.

### Bug Description

Issue in the message of login page under password field
It should not say what length password it is

### Screenshots

![image](https://github.com/user-attachments/assets/2125dc73-44cc-46db-9c28-623789078e26)

### Would you like to work on this issue?

Yes

### Implementation Plan

I will change the error message

**Comment by ajaynegi45** [comment]
> ### Is this a new issue?
> * [x]  I have searched "open" and "closed" issues, and this is not a duplicate.
> 
> ### Bug Description
> Issue in the message of login page under password field It should not say what length password it is
> 
> ### Steps to Reproduce
> _No response_
> 
> ### Screenshots
> ![image](https://private-user-images.githubusercontent.com/114927759/377055052-2125dc73-44cc-46db-9c28-623789078e26.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwODQ3MDQsIm5iZiI6MTcyOTA4NDQwNCwicGF0aCI6Ii8xMTQ5Mjc3NTkvMzc3MDU1MDUyLTIxMjVkYzczLTQ0Y2MtNDZkYi05YzI4LTYyMzc4OTA3OGUyNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNlQxMzEzMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZDgxZmM0NTI0ZDM1MjE2OTgyNjE2NzUwODU0NWYxMWY0ODQzMmY3OGYwNTUwYTI2OGNjYmY5NWI4NzFiNjRhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.I3VMZN5RRMhnTTa4ytjMIn6W43dEVWJzeuJpuYq5n5g)
> 
> ### Would you like to work on this issue?
> Yes
> 
> ### Implementation Plan
> I will change the error message

Make like this dynamic. https://medium.com/@sharathchandark/password-validation-check-in-html-css-javascript-show-hide-password-toggle-c57ebde8eb8e

**Comment by ajaynegi45** [comment]
Hi @Gauravtb2253,

Hope you're doing great! 😊 Just wanted to check in and see how things are coming along with the issue you're working on. Any progress or updates you’d like to share? And if you’ve run into any roadblocks or need a hand with anything, please don’t hesitate to reach out!

Looking forward to hearing from you!

**Comment by sameer-soni** [comment]
@ajaynegi45 looks like theres no update, can i get assigned? I'll solve this quickly

**Comment by ajaynegi45** [comment]
> @ajaynegi45 looks like theres no update, can i get assigned? I'll solve this quickly

Yes, you can work on this and correct the regex as well, because currently it only takes one special character.

**Comment by sameer-soni** [comment]
@ajaynegi45 done. 

https://github.com/user-attachments/assets/9f96dd4c-ead0-4382-b3ed-028d4c509c4f

**Comment by ajaynegi45** [comment]
> @ajaynegi45 done.
> 
>  Untitled.video.-.Made.with.Clipchamp.5.mp4

Hide them by default. When the password field is active then show them
